src/tests/multihost/alltests/test_misc.py

```diff
@@ -225,3 +224,122 @@ def test_0005_getent_homedirectory(self, multihost,
+        :title: 'getent group ldapgroupname' doesn't
+         show any LDAP users or some LDAP users when
+        'rfc2307bis' schema is used with SSSD
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1817122
+        :id: dc81bb8e-72c0-11eb-9eae-002b677efe14
+        :steps:
+            1. Configure SSSD with id_provider = ldap and
+            set ldap_schema = rfc2307bis
+            2. Check 'id ldapusername' output.
+            3. Check 'getent group ldapgroupname' output.
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. 'getent group ldapgroupname' should show
+            all it's member ldapusers.
+        """
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        client = sssdTools(multihost.client[0])
+        domain_params = {'ldap_schema': 'rfc2307bis',
+                         'ldap_group_member': 'uniquemember'}
+        client.sssd_conf(f'domain/{domain_name}', domain_params)
+        multihost.client[0].service_sssd('restart')
+        ldap_uri = 'ldap://%s' % (multihost.master[0].sys_hostname)
+        ds_rootdn = 'cn=Directory Manager'
+        ds_rootpw = 'Secret123'
+        ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
+        user_info = {
+            'ou': 'Unit1'.encode('utf-8'),
+            'objectClass': [b'top', b'organizationalUnit']}
+        user_dn = 'ou=Unit1,dc=example,dc=test'
+        (_, _) = ldap_inst.add_entry(user_info, user_dn)
+        user_info = {
+            'ou': 'Unit2'.encode('utf-8'),
+            'objectClass': [b'top', b'organizationalUnit']}
+        user_dn = 'ou=Unit2,ou=Unit1,dc=example,dc=test'
+        (_, _) = ldap_inst.add_entry(user_info, user_dn)
+        user_info = {
+            'ou': 'users'.encode('utf-8'),
+            'objectClass': [b'top', b'organizationalUnit']}
+        user_dn = 'ou=users,ou=Unit2,ou=Unit1,dc=example,dc=test'
+        (_, _) = ldap_inst.add_entry(user_info, user_dn)
+        user_info = {
+            'ou': 'posix_groups'.encode('utf-8'),
+            'objectClass': [b'top', b'organizationalUnit']}
+        user_dn = 'ou=posix_groups,ou=Unit2,' \
+                  'ou=Unit1,dc=example,dc=test'
+        (_, _) = ldap_inst.add_entry(user_info, user_dn)
+        user_info = {
+            'ou': 'netgroups'.encode('utf-8'),
+            'objectClass': [b'top', b'organizationalUnit']}
+        user_dn = 'ou=netgroups,dc=example,dc=test'
+        (_, _) = ldap_inst.add_entry(user_info, user_dn)
+        user_info = {
+            'ou': 'services'.encode('utf-8'),
+            'objectClass': [b'top', b'organizationalUnit']}
+        user_dn = 'ou=services,dc=example,dc=test'
+        (_, _) = ldap_inst.add_entry(user_info, user_dn)
+        user_info = {
+            'ou': 'sudoers'.encode('utf-8'),
+            'objectClass': [b'top', b'organizationalUnit']}
+        user_dn = 'ou=sudoers,dc=example,dc=test'
```

Comment:
Do we need netgroups, services and sudoers OUs to be created for this test?

Reply:
yes , we need these